package.json

```diff
@@ -77,58 +77,57 @@
   },
   "homepage": "https://github.com/allusion-app/Allusion#readme",
   "devDependencies": {
-    "@svgr/webpack": "^5.5.0",
+    "@svgr/webpack": "^6.1.2",
     "@types/chrome": "^0.0.86",
     "@types/fs-extra": "^9.0.0",
-    "@types/jest": "^26.0.0",
+    "@types/jest": "^27.4.0",
     "@types/offscreencanvas": "^2019.6.2",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.3",
-    "@types/react-window": "^1.8.2",
+    "@types/react": "^17.0.38",
+    "@types/react-dom": "^17.0.11",
+    "@types/react-window": "^1.8.5",
     "@types/utif": "^3.0.1",
     "@types/uuid": "^8.3.0",
-    "@typescript-eslint/eslint-plugin": "^4.2.0",
-    "@typescript-eslint/parser": "^4.2.0",
-    "css-loader": "^5.2.0",
+    "@typescript-eslint/eslint-plugin": "^5.9.0",
+    "@typescript-eslint/parser": "^5.9.0",
+    "css-loader": "^6.5.1",
     "electron": "^15.3.3",
     "electron-builder": "^22.10.5",
-    "eslint": "^7.18.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-react": "^7.22.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "html-webpack-plugin": "^5.3.1",
-    "jest": "^26.6.3",
-    "mini-css-extract-plugin": "^1.4.1",
+    "eslint": "^8.6.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.28.0",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "html-webpack-plugin": "^5.5.0",
+    "jest": "^27.4.7",
+    "mini-css-extract-plugin": "^2.4.6",
     "ncp": "^2.0.0",
-    "node-loader": "^2.0.0",
```

Comment:
Did we ever need it?

Reply:
I tried removing it once before, it broke the build for MacOS because `fsevents` depends on it 
https://github.com/allusion-app/Allusion/runs/4279170267